web/packages/teleterm/src/ui/Search/SearchBar.test.tsx

```diff
@@ -323,7 +323,6 @@ const getMockedSearchContext = (): SearchContext.SearchContext => ({
   isOpen: true,
   open: () => {},
   close: () => {},
-  closeAndResetInput: () => {},
```

Comment:
In the upcoming commits I'm going to add `closeWithoutRestoringFocus`. I didn't want to have three different `close` variants and I didn't want to have `close({restoreFocus: boolean, resetInput: boolean})` either. So I decided to get rid of `closeAndResetInput` as it was used only in a single specific situation which already also called `resetInput` anyway.